doozer/doozerlib/cli/release_gen_payload.py

```diff
@@ -150,14 +150,17 @@ async def release_gen_payload(runtime: Runtime, is_name: str, is_namespace: str,
     ).run()
 
 
-def default_imagestream_base_name(version: str) -> str:
-    return f"{version}-art-latest"
+def default_imagestream_base_name(version: str, runtime: Runtime) -> str:
+    if runtime.build_system == 'brew':
+        return f"{version}-art-latest"
+    else:  # konflux
```

Comment:
nitpik but should we assert build_system is not `None` / something other than konflux? or maybe default to brew imagestream value

Reply:
`build_system` is not None by design, and defaults to `'brew'` to make sure we don't alter existing functionalities unless intended: https://github.com/openshift-eng/art-tools/blob/main/doozer/doozerlib/cli/__init__.py#L107